File: bubble-sort.py
#

def bubbleSort(arr):
    for i in range(len(arr)):
        for j in range(0, len(arr)-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr

arr = [5,1,4,2,8]
ordered = bubbleSort(arr)
print(ordered)

File: bubble-sort.js
function bubbleSort(arr) {
	for(let x=0; x<arr.length; x++) {
		for(let y=0; y<arr.length-x-1; y++) {
			if(arr[y] > arr[y+1]) {
				[ arr[y], arr[y+1] ] = [ arr[y+1], arr[y] ];
			}
		}
	}
	return arr;
}

const arr = [5,1,4,2,8];
console.log(bubbleSort(arr));
